include/envoy/upstream/upstream.h

```diff
@@ -95,6 +95,11 @@ class Host : virtual public HostDescription {
    * Set the current load balancing weight of the host, in the range 1-100.
    */
   virtual void weight(uint32_t new_weight) PURE;
+
+  /**
+   * @return the old value of host being in use and set the value to 'new_used'.
```

Comment:
Fixed comment, splitting to a setter/getter pair will have too wait for tomorrow.